server/src/util/file-util.ts

```diff
@@ -0,0 +1,44 @@
+import { default as fsWithCallbacks } from "fs";
+import path from "path";
+import { default as unzipper } from "unzipper";
+import { devAssert } from "./development-util";
+const fs = fsWithCallbacks.promises;
+
+
+
+export async function unzip(sourcePath: string, targetPath: string): Promise<void> {
+  devAssert(() => fsWithCallbacks.existsSync(sourcePath), "Sourcepath does not exist.");
+  devAssert(() => !fsWithCallbacks.existsSync(targetPath), "Targetpath already exists.");
+
+  const stream = fsWithCallbacks.createReadStream(sourcePath)
+    .pipe(unzipper.Extract({ path: targetPath }));
```

Comment:
Express-fileupload doesn't directly support this feature. If we want to skip the zip file itself, then we'd have to find a different library. 
However, I feel like it might be useful to have the zip file around for reference.